docs/source/conf.py

```diff
@@ -145,14 +147,26 @@
 autoapi_template_dir = "_templates/autoapi"
 
 
+suppress_warnings = [
+    "autosectionlabel.*",
+]
+
+
 def skip_log(app, what, name, obj, skip, options):
     if what == "data" and name.split(".")[-1] == "log":
         skip = True
     return skip
 
 
 def setup(sphinx):
-    sphinx.connect("autoapi-skip-member", skip_log)
+    """"""
+    # sphinx.connect("autoapi-skip-member", skip_log)
```

Comment:
Why remove that skipping, we wanted to not show the logger everywhere

Reply:
Temporarily disabled autoapi during development (it takes a long time to build). Thanks for catching this.